### PR TITLE
opentrack: Use portable configuration

### LINKED ISSUE #3548: opentrack: Use portable configuration
https://github.com/opentrack/opentrack/wiki/portable-mode-for-USB-sticks

Migrate or add new `-portable` manifest.

### PR DESCRIPTION
**Pull request by Lutra-Fs**
Closes #3548 

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
- Is `ini` a folder?
- How has the PR been tested?

**Comment by Lutra-Fs** [comment]
- Yes, `ini` is a folder. By creating `portable.txt`, the user config will be stored in this folder under the installation path.
- I have tested the PR on my own device.

File: bucket/opentrack.json [hunk 1]
```diff
@@ -13,6 +13,17 @@
             "Opentrack"
         ]
     ],
+    "post_install": [
+        "if (!(Test-Path \"$dir\\portable.txt\")) {",
+        "    info '[Portable Mode] Adding Portable Mode File...'",
+        "    New-Item \"$dir\\portable.txt\" -ItemType File",
```
Comment (rashil2000) [inline bucket/opentrack.json:19]:
```suggestion
        "    New-Item \"$dir\\portable.txt\" -ItemType File | Out-Null",
```

File: bucket/opentrack.json [hunk 2]
```diff
@@ -13,6 +13,17 @@
...
+        "}",
+        "if (!(Test-Path \"$dir\\ini\\*\") -and ([Environment]::GetFolderPath(\"MyDocuments\")+\"\\opentrack-2.3\" | Test-Path)) {",
+        "    info '[Portable Mode] Copying User Settings...'",
+        "     [Environment]::GetFolderPath(\"MyDocuments\")+\"\\opentrack-2.3\\*\" | Copy-Item -destination \"$dir\\ini\" -Recurse",
```
Comment (rashil2000) [inline bucket/opentrack.json:23]:
```suggestion
        "    [Environment]::GetFolderPath(\"MyDocuments\")+\"\\opentrack-2.3\\*\" | Copy-Item -Destination \"$dir\\ini\" -Recurse",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**